Resources/Prototypes/_Ronstation/Entities/Objects/Specific/deckardbox.yml

```
@@ -0,0 +1,63 @@
+- type: entity
+  id: DeckardBox
+  parent: [ BaseStorageItem, BaseBagOpenClose ]
+  name: deckard box
+  description: A heavy wooden box, specially designed to hold a collectible revolver and its speedloaders.
```

```suggestion
  description: An artisanal wooden box made to house the Deckard and two speed loaders. It has a good weight to it.
```
I feel like this reads better but I tend to overthink item descriptions, lol.